data/scenarios/scenarios.yaml

```diff
@@ -0,0 +1,27 @@
+# ------------------------------------------------------------------------
+#                       -- Scenarios --
+# ------------------------------------------------------------------------
+#
+# This is a config file to define scenarios.
+#
+# Each scenario consists of the following fields:
+#
+#                       -- metadata --
+#       display_name        Simple string that contains the display name that
+#                           be used in an UI to describe the question.
+#
+#       description         Further description of the question, what the
+#                           question attempts to answer, whether it will
+#                           require further manual analysis, etc.
+#
+# ------------------------------------------------------------------------
+compromise_assessment:
+  display_name: Compromise Assessment
+  description: Maecenas tristique sapien lorem, et vehicula ipsum vestibulum
```

Comment:
We should have a good description here or add an TODO

Reply:
Yes, these placeholders need to be updated in a future PR :)